deploy/helm_charts/envs/mixer_encode.yaml

```diff
@@ -26,5 +26,6 @@ serviceGroups:
 ip: 34.107.228.109
 region: us-central1
 api_title: DataCommons API (Encode)
-api: datacommons.Mixer
+api:
+  - datacommons.Mixer
```

Comment:
just for my understanding - is this purely a formatting change, or does the `-` have an effect?

Reply:
This represents a list, while the old way should be a single value https://docs.ansible.com/ansible/latest/reference_appendices/YAMLSyntax.html#yaml-basics